src/java.base/share/classes/java/util/Arrays.java

```diff
@@ -4166,6 +4177,11 @@ public E get(int index) {
             return a[index];
         }
 
+        /**
+         * {@inheritDoc}
+         *
+         * @throws ArrayStoreException if {@code element} cannot be stored into the array.
+         */
```

Comment:
No javadoc is ever produced for this internal  implementation class, so doc comments here are not needed IMO.